src/sentry/static/sentry/app/views/settings/projectProguard/projectProguardRow.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Access from 'app/components/acl/access';
+import Role from 'app/components/acl/role';
+import Button from 'app/components/button';
+import ButtonBar from 'app/components/buttonBar';
+import Confirm from 'app/components/confirm';
+import FileSize from 'app/components/fileSize';
+import TimeSince from 'app/components/timeSince';
+import Tooltip from 'app/components/tooltip';
+import {IconClock, IconDelete, IconDownload} from 'app/icons';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {DebugFile} from 'app/types/debugFiles';
+
+type Props = {
+  mapping: DebugFile;
+  onDelete: (id: string) => void;
+  downloadUrl: string;
+  downloadRole: string;
+};
+
+const ProjectProguardRow = ({mapping, onDelete, downloadUrl, downloadRole}: Props) => {
+  const {id, debugId, uuid, size, dateCreated} = mapping;
+
+  const handleDeleteClick = () => {
+    onDelete(id);
+  };
+
+  return (
+    <React.Fragment>
+      <NameColumn>
+        <Name>{debugId || uuid || `(${t('empty')})`}</Name>
+        <TimeWrapper>
+          <IconClock size="sm" />
+          <TimeSince date={dateCreated} />
+        </TimeWrapper>
+      </NameColumn>
+      <SizeColumn>
+        <FileSize bytes={size} />
+      </SizeColumn>
+      <ActionsColumn>
```

Comment:
should we just use `SizeColumn` here?

Reply:
We could, I felt like this was more semantic 🤷‍♂️